### ISSUE TITLE
400 bad request downloading tariff details

### ISSUE THREAD

**Issue by NiccyB** [hypercrypt/octopus-db#1]
Running 'php octopus.php' gives the following error:
```
PHP Warning:  Undefined array key 0 in /home/nicholas/octopus-db/classes/OctopusEnergy.php on line 35
PHP Warning:  Attempt to read property "meters" on null in /home/nicholas/octopus-db/classes/OctopusEnergy.php on line 35
PHP Warning:  foreach() argument must be of type array|object, null given in /home/nicholas/octopus-db/classes/OctopusEnergy.php on line 35
PHP Fatal error:  Uncaught GuzzleHttp\Exception\ClientException: Client error: `GET https://api.octopus.energy/v1/products/GB-FIXED-SEPTEMBER-20-ELEC-2-RATE/electricity-tariffs/E-2R-GB-FIXED-SEPTEMBER-20-ELEC-2-RATE-B/standard-unit-rates/?page=1&page_size=100` resulted in a `400 Bad Request` response:
{"detail":"This tariff has day and night rates, not standard."}
 in /home/nicholas/octopus-db/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:113
Stack trace:
#0 /home/nicholas/octopus-db/vendor/guzzlehttp/guzzle/src/Middleware.php(69): GuzzleHttp\Exception\RequestException::create()
#1 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/Promise.php(204): GuzzleHttp\Middleware::GuzzleHttp\{closure}()
#2 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/Promise.php(153): GuzzleHttp\Promise\Promise::callHandler()
#3 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/TaskQueue.php(48): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}()
#4 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/Promise.php(248): GuzzleHttp\Promise\TaskQueue->run()
#5 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/Promise.php(224): GuzzleHttp\Promise\Promise->invokeWaitFn()
#6 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/Promise.php(269): GuzzleHttp\Promise\Promise->waitIfPending()
#7 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/Promise.php(226): GuzzleHttp\Promise\Promise->invokeWaitList()
#8 /home/nicholas/octopus-db/vendor/guzzlehttp/promises/src/Promise.php(62): GuzzleHttp\Promise\Promise->waitIfPending()
#9 /home/nicholas/octopus-db/vendor/guzzlehttp/guzzle/src/Client.php(187): GuzzleHttp\Promise\Promise->wait()
#10 /home/nicholas/octopus-db/vendor/guzzlehttp/guzzle/src/ClientTrait.php(44): GuzzleHttp\Client->request()
#11 /home/nicholas/octopus-db/classes/OctopusEnergy.php(48): GuzzleHttp\Client->get()
#12 /home/nicholas/octopus-db/classes/OctopusEnergy.php(72): OctopusEnergy->fetch_json()
#13 /home/nicholas/octopus-db/octopus.php(125): OctopusEnergy->get_prices()
#14 {main}
  thrown in /home/nicholas/octopus-db/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php on line 113
```

**Comment by hypercrypt** [comment]
I’ll get support for E7 added ASAP

**Comment by hypercrypt** [comment]
Okay, I see the issue - looks like I need to add a bit more support for E7 so may take a little bit of time but hope to get this added before Christmas

**Comment by klaus-gsma** [comment]
This should be resolved in v0.0.2

**Comment by hypercrypt** [comment]
This should be resolved in v0.0.2
